<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="DownloadQueueView" name="downloadQueueView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionImportWizard"/>
    <addaction name="separator"/>
    <addaction name="actionImportFromFile"/>
    <addaction name="actionExportSelectedToFile"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionSpeedLimit"/>
    <addaction name="actionAddDomainSpecificLimit"/>
    <addaction name="separator"/>
    <addaction name="actionForceStart"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuOther1">
     <property name="title">
      <string>Other</string>
     </property>
     <addaction name="actionRemoveAll1"/>
     <addaction name="separator"/>
     <addaction name="actionCleanGoneFiles"/>
    </widget>
    <widget class="QMenu" name="menuOther2">
     <property name="title">
      <string>Other</string>
     </property>
     <addaction name="actionRemoveSelected"/>
     <addaction name="separator"/>
     <addaction name="actionRemoveDuplicates"/>
     <addaction name="separator"/>
     <addaction name="actionRemoveAll2"/>
     <addaction name="separator"/>
     <addaction name="actionRemoveFailed"/>
     <addaction name="actionRemovePaused"/>
    </widget>
    <addaction name="actionInformation"/>
    <addaction name="separator"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionRenameFile"/>
    <addaction name="actionDeleteFile"/>
    <addaction name="actionOpenDirectory"/>
    <addaction name="separator"/>
    <addaction name="actionRemoveCompletedDownloads"/>
    <addaction name="menuOther1"/>
    <addaction name="separator"/>
    <addaction name="actionRemoveDownloads"/>
    <addaction name="menuOther2"/>
   </widget>
   <widget class="QMenu" name="menuDownload">
    <property name="title">
     <string>Download</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="separator"/>
    <addaction name="actionResume"/>
    <addaction name="actionPause"/>
    <addaction name="actionCancel"/>
    <addaction name="separator"/>
    <addaction name="actionTop"/>
    <addaction name="actionUp"/>
    <addaction name="actionDown"/>
    <addaction name="actionBottom"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionSelectAll"/>
    <addaction name="actionSelectNone"/>
    <addaction name="actionInvertSelection"/>
    <addaction name="actionSelectCompleted"/>
    <addaction name="separator"/>
    <addaction name="actionManageMirrors"/>
    <addaction name="actionOneMoreSegment"/>
    <addaction name="actionOneFewerSegment"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuDownload"/>
   <addaction name="menuOption"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="lateralBar">
   <property name="windowTitle">
    <string>File toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInformation"/>
   <addaction name="separator"/>
   <addaction name="actionOpenFile"/>
   <addaction name="actionDeleteFile"/>
   <addaction name="actionOpenDirectory"/>
   <addaction name="separator"/>
   <addaction name="actionSelectAll"/>
   <addaction name="actionInvertSelection"/>
   <addaction name="separator"/>
   <addaction name="actionManageMirrors"/>
   <addaction name="actionOneMoreSegment"/>
   <addaction name="actionOneFewerSegment"/>
   <addaction name="separator"/>
   <addaction name="actionForceStart"/>
   <addaction name="separator"/>
   <addaction name="actionImportFromFile"/>
   <addaction name="actionExportSelectedToFile"/>
  </widget>
  <widget class="QToolBar" name="downloadBar">
   <property name="windowTitle">
    <string>View toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImportWizard"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="separator"/>
   <addaction name="actionResume"/>
   <addaction name="actionPause"/>
   <addaction name="actionCancel"/>
   <addaction name="separator"/>
   <addaction name="actionTop"/>
   <addaction name="actionUp"/>
   <addaction name="actionDown"/>
   <addaction name="actionBottom"/>
   <addaction name="separator"/>
   <addaction name="actionPreferences"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/button_help_16x16.png</normaloff>:/icons/menu/button_help_16x16.png</iconset>
   </property>
   <property name="text">
    <string>About DownZemAll...</string>
   </property>
   <property name="toolTip">
    <string>About DownZemAll</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/button_preference_16x16.png</normaloff>:/icons/menu/button_preference_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/tool_add_26x26.png</normaloff>:/icons/menu/tool_add_26x26.png</iconset>
   </property>
   <property name="text">
    <string>Add...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_cancel_16x16.png</normaloff>:/icons/menu/icon_cancel_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_pause_16x16.png</normaloff>:/icons/menu/icon_pause_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionUp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_up_16x16.png</normaloff>:/icons/menu/icon_up_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
   <property name="shortcut">
    <string>Alt+PgUp</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_top_16x16.png</normaloff>:/icons/menu/icon_top_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
   <property name="shortcut">
    <string>Alt+Home</string>
   </property>
  </action>
  <action name="actionDown">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_down_16x16.png</normaloff>:/icons/menu/icon_down_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
   <property name="shortcut">
    <string>Alt+PgDown</string>
   </property>
  </action>
  <action name="actionBottom">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_bottom_16x16.png</normaloff>:/icons/menu/icon_bottom_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Bottom</string>
   </property>
   <property name="shortcut">
    <string>Alt+End</string>
   </property>
  </action>
  <action name="actionResume">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_resume_16x16.png</normaloff>:/icons/menu/icon_resume_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Resume</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_info_16x16.png</normaloff>:/icons/menu/icon_info_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Download Information</string>
   </property>
   <property name="shortcut">
    <string>Alt+I</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_open_file_16x16.png</normaloff>:/icons/menu/icon_open_file_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionRenameFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_rename_file_16x16.png</normaloff>:/icons/menu/icon_rename_file_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Rename File</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionDeleteFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_delete_file_16x16.png</normaloff>:/icons/menu/icon_delete_file_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Delete File(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionOpenDirectory">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_open_directory_16x16.png</normaloff>:/icons/menu/icon_open_directory_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Open Directory</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/select_all_32x32.png</normaloff>:/icons/menu/select_all_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionInvertSelection">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/select_invert_32x32.png</normaloff>:/icons/menu/select_invert_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Invert Selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionManageMirrors">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_manage_mirror_48x48.png</normaloff>:/icons/menu/icon_manage_mirror_48x48.png</iconset>
   </property>
   <property name="text">
    <string>Manage Download Mirror Locations...</string>
   </property>
   <property name="toolTip">
    <string>Manage Download Mirror Locations</string>
   </property>
  </action>
  <action name="actionOneMoreSegment">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_segment_add_16x16.png</normaloff>:/icons/menu/icon_segment_add_16x16.png</iconset>
   </property>
   <property name="text">
    <string>One More Segment</string>
   </property>
  </action>
  <action name="actionOneFewerSegment">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_segment_remove_16x16.png</normaloff>:/icons/menu/icon_segment_remove_16x16.png</iconset>
   </property>
   <property name="text">
    <string>One Fewer Segment</string>
   </property>
  </action>
  <action name="actionForceStart">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_force_start_16x16.png</normaloff>:/icons/menu/icon_force_start_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Force Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionImportFromFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_import_from_file_16x16.png</normaloff>:/icons/menu/icon_import_from_file_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Import From File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionExportSelectedToFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_export_selected_to_file_16x16.png</normaloff>:/icons/menu/icon_export_selected_to_file_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Export &amp;Selected To File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S, Ctrl+S</string>
   </property>
  </action>
  <action name="actionRemoveCompletedDownloads">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_remove_completed_16x16.png</normaloff>:/icons/menu/icon_remove_completed_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Remove Completed Downloads</string>
   </property>
  </action>
  <action name="actionRemoveDownloads">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_remove_downloaded_16x16.png</normaloff>:/icons/menu/icon_remove_downloaded_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Remove Downloads</string>
   </property>
  </action>
  <action name="actionCleanGoneFiles">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_clean_gone_file_16x16.png</normaloff>:/icons/menu/icon_clean_gone_file_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Clean Gone Files</string>
   </property>
   <property name="toolTip">
    <string>Clean Gone Files</string>
   </property>
   <property name="shortcut">
    <string>Alt+Del</string>
   </property>
  </action>
  <action name="actionRemoveDuplicates">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_remove_duplicates_16x16.png</normaloff>:/icons/menu/icon_remove_duplicates_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Remove Duplicates</string>
   </property>
  </action>
  <action name="actionAddDomainSpecificLimit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_add_domain_specific_limit_16x16.png</normaloff>:/icons/menu/icon_add_domain_specific_limit_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Add Domain Specific Limit...</string>
   </property>
  </action>
  <action name="actionSpeedLimit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_speed_limit_16x16.png</normaloff>:/icons/menu/icon_speed_limit_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Speed Limit...</string>
   </property>
   <property name="toolTip">
    <string>Speed Limit...</string>
   </property>
  </action>
  <action name="actionSelectNone">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/select_none_32x32.png</normaloff>:/icons/menu/select_none_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Select None</string>
   </property>
   <property name="toolTip">
    <string>Select None</string>
   </property>
  </action>
  <action name="actionSelectCompleted">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/select_completed_32x32.png</normaloff>:/icons/menu/select_completed_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Select Completed</string>
   </property>
  </action>
  <action name="actionImportWizard">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/wizard_add_16x16.png</normaloff>:/icons/menu/wizard_add_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Import Wizard... </string>
   </property>
   <property name="toolTip">
    <string>Import Wizard  (Ctrl+X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionRemoveAll1">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_remove_completed_16x16.png</normaloff>:/icons/menu/icon_remove_completed_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Remove All</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="actionRemoveSelected">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_remove_downloaded_16x16.png</normaloff>:/icons/menu/icon_remove_downloaded_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Remove Selected</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionRemoveFailed">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_cancel_16x16.png</normaloff>:/icons/menu/icon_cancel_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Remove Failed</string>
   </property>
  </action>
  <action name="actionRemovePaused">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_pause_16x16.png</normaloff>:/icons/menu/icon_pause_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Remove Paused</string>
   </property>
  </action>
  <action name="actionRemoveAll2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/icon_remove_downloaded_16x16.png</normaloff>:/icons/menu/icon_remove_downloaded_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Remove All</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DownloadQueueView</class>
   <extends>QWidget</extends>
   <header>Widgets/DownloadQueueView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
